assets/src/components/nav/topNav.tsx

```diff
@@ -19,6 +30,32 @@ const TopNav = (): JSX.Element => {
             <RefreshIcon className="c-top-nav__icon" />
           </button>
         </li>
+        {showLoggedInUser && (
+          <li>
+            <div ref={userButtonRef}>
+              <CircleButton
+                isActive={showUserPopover}
+                onClick={() => {
```

Comment:
question: I saw in the designs that it called for a `Dropdown`, is this the correct component to be using

Reply:
This comment was intended on this range >.>
https://github.com/mbta/skate/pull/2385/files/8a88981ec38aff4be22add67a03ea4da807e655c#diff-d9e5ec15cc9d903f89bcbd4876846b898ec74d0027a066930b7fe42d93f259f8R46-R56

Reply:
Mariele made a comment at one point that while it said "dropdown", it was actually supposed to be a bootstrap ... _something else_, and I don't remember exactly what she said.

I went with the component that looked the most like the design.

Reply:
Sounds good! I was surprised that the links on the components in figma linked to the dropdown docs, but looked like a popover.